x/wasm/simulation/operations.go

```diff
@@ -0,0 +1,163 @@
+package simulation
+
+import (
+	_ "embed"
+	"math/rand"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/CosmWasm/wasmd/app/params"
+	"github.com/CosmWasm/wasmd/x/wasm/types"
+)
+
+//go:embed testdata/reflect.wasm
+var reflectContract []byte
```

Comment:
Would it be possible to embed from the keeper testdata so that we do not have to maintain copies of the file?